addons/medical_treatment/functions/fnc_getTriageStatus.sqf

```diff
@@ -14,10 +14,10 @@
  * Public: Yes
  */
 
-private ["_unit","_return","_status"];
 params ["_unit"];
-_status = _unit getVariable [QEGVAR(medical,triageLevel), -1];
-_return = switch (_status) do {
+
+private _status = _unit getVariable [QEGVAR(medical,triageLevel), -1];
+private _return = switch (_status) do {
```

Comment:
The use of a return variable is pointless in this function since the switch is at the end anyway 👍

Reply:
How about replacing the switch by an `ARRAY param [_status, default]` ?

Reply:
Nah. Would then call `localize` for all entries everytime